package.json

```diff
@@ -54,6 +54,7 @@
   "scripts": {
     "prepublish": "gulp prepublish",
     "pretest": "gulp pretest",
+    "prestart": "webdriver-manager start > /dev/null 2>&1 &",
```

Comment:
Awesome! So, I think that is a better way if you close this PR, because it's showing my last commit that was already merged in another PR, so, if you close this one I'll open again with the new solution for this, Could u do that? :)